### PR TITLE
Dump: Fix .name/.property doublettes

### LINKED ISSUE #750: Dump: .name/.message-property doublettes
If an Object has the properties .name and/or .message the dump of the object contains a doublette of said properties.
Example: http://jsfiddle.net/c50ndqhq/1/

### PR DESCRIPTION
**Pull request by bubens**
Fix #750

### CODE REVIEW

File: src/dump.js [hunk 1]
```diff
@@ -174,7 +174,7 @@ QUnit.dump = (function() {
 					nonEnumerableProperties = [ "message", "name" ];
 					for ( i in nonEnumerableProperties ) {
 						key = nonEnumerableProperties[ i ];
-						if ( key in map && !( key in keys ) ) {
+						if ( key in map && keys.indexOf( key ) < 0 ) {
```
Comment (leobalter) [inline src/dump.js]: indexOf is [IE9+](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/indexOf)

**Comment by bubens** [comment]
Changed it. Now using the internal inArray() function to test for key in keys. Since this is an internal function it should meet all the browser-compatibilty requirements.

P.S.: This is the first PR I did and I'm just an amateur doing some coding at home. I hope it helps and is not causing any or even too much trouble.

**Comment by jzaefferer** [comment]
Thank you for the contribution! We still need you to sign our CLA though: http://contribute.jquery.org/CLA/

Once that is done, this should be good to land.

**Comment by bubens** [comment]
Signed it.

**Comment by gpascale** [comment]
Hey, can you guys merge this in? I keep hitting this issue. Was just about to submit a fix, but then I saw @bubens had already gotten to it.

**Comment by leobalter** [comment]
@jzaefferer I would land this, but the CLA check is failing here.

```
Email is Valid ( bubens@unpunk.de )
Name is at least 2 parts ( bubens )
Confirmed signing
Name does not match bubens != Henning Beyer
```

**Comment by bubens** [comment]
The name that didn't match (Henning Beyer) is me. Sorry, new to github. Changed my profile so that the name should match now.

**Comment by leobalter** [comment]
The CLA check is failing due to the commit's author name.

Maybe you might need to reset your name with:

```bash
git config --global user.name "YOUR NAME"
```
and then, squash your commits into a single one, and reset the author name to match your full name, than push it again to this branch.

```
git commit --amend --reset-author
git push your-remote +name-message-property-doublettes
```

@arthurvr and @scottgonzalez, would you help me on this?

**Comment by scottgonzalez** [comment]
@leobalter The check says it's valid.

**Comment by scottgonzalez** [comment]
Hmm...though the name isn't valid. I guess we're only doing name validation on the signatures, not on the commit. I've filed https://github.com/jquery/jquery-license/issues/25 for that issue.

@leobalter You can just do the squash and author change yourself. It's often easier for us to fix when landing than for new contributors to fix on their own.

**Comment by leobalter** [comment]
Ok, I'll do it.

**Comment by bubens** [comment]
I'm sorry for the mess. I read all the howtos and guidelines but missed this one. All this for not even a single line of code...
Thanks for your patience.

**Comment by leobalter** [comment]
No problem, CLAs are legally necessary for jQuery as a foundation and they require some strict rules on commit messages that might be not friendly sometimes.

**Pull request closed without merging**